tasks/setup-Debian.yml

```diff
@@ -32,12 +32,14 @@
   apt:
     name: "{{ java_packages }}"
     default_release: stretch-backports
+    update_cache: true
```

Comment:
`cache_valid_time` could be used instead of `update_cache`.
Note that:
1. `update_cache` parameter of `apt_repository` Ansible module is enabled by default
2. `apt-daily.service` is triggered by `apt-daily.timer` every day - this is disabled by default in Debian but enabled by [peopleinfra-base-configuration](https://github.com/peopledoc/peopleinfra-base-configuration/blob/225d81ede679ad8470767e3c02e778b1cc5058e3/defaults/main.yml#L4))

Reply:
`cache_valid_time` is not a parameter of  `apt_repository`. I removed `update_cache` for those tasks.